client/main.js

```diff
@@ -106,9 +106,9 @@ $(document).ready(function() {
     // About
     window.location = '/about';
   });
-  window.Mousetrap.bind('g n s', () => {
-    // Shop
-    window.location = '/shop';
+  window.Mousetrap.bind('g n d', () => {
+    // Donate
+    window.location = 'https://www.freecodecamp.com/donate';
```

Comment:
I'd like to see this string added to https://github.com/freeCodeCamp/freeCodeCamp/blob/staging/common/utils/constantStrings.json

and imported from there so it's easier to refactor at some later time